pixi/src/ui/I18n.js

```diff
@@ -45,7 +61,7 @@ class I18n {
         const {body, ...props} = recommendation;
         result.push({
           id,
-          body: body || item.description,
+          body: body || marked(item.description),
```

Comment:
Where does `item.description` come from? Can we not easily convert this on the server as well?

Reply:
That's coming from the Page Speed Insights API... I thought of multiple solutions, none of them really clever: a) round-trip to an amp.dev API to convert, b) proxy PSI API somewhere and convert response there, c) do not rely on the PSI texts but mirror them inside the amp.dev project

Any other ideas?

Reply:
something close to (b) makes sense to me: have our api also make the psi request

Reply:
By rendering the html on the server side we only fixed the problem that the apostrophe and the quote signs where not shown as entities. Since we do not use the HTML inside of an attribute we do not need to use entities at all, so simply converting them back on the client also works for us.
So I see no benefit of converting the PSI request on the server side aside from having smaller JS code on the client side. In each case we still have the problem that the code examples in some recommendations which use &lt; and &gt; are not shown correctly. Because of this issue ampproject/amphtml#27153 all html entities we have in the html code will be shown as entities on the site and not as their respective character.
That is why I also replaced those entities with other unicode characters that look the same. It's just an ugly workaround. Without some improvement on the amp side regarding the mentioned issue I see no possibility to display all our recommendations correctly without really rendering the recommendations on the server side and not using DOM manipulation to add them.